### PR TITLE
Recipe list v2

### LINKED ISSUE #19: Create/Manage Recipe List

### PR DESCRIPTION
**Pull request by weix3**
I create the new component named recipe-list.

### CODE REVIEW

**Comment by weix3** [comment]
-pull request for "Create/Manage Recipe List"

**Review by callumhauber — APPROVED** [review]
Nice work. It looks a little weird without the sidebar but I'm waiting until these are all merged in to see what it looks like

**Pull request merged**
